frontend/public/locales/en/chat.json

```diff
@@ -12,7 +12,7 @@
     "receivedResponse": "Received response. Starting to handle stream...",
     "errorCallingAPI": "Error calling the API",
     "ask": "Ask a question, or describe a task.",
-    "begin_conversation_placeholder": "Begin conversation here...",
+    "begin_conversation_placeholder": "Begin conversation here, @ to select a brain...",
```

Comment:
Is this actually possible now?

It should be introduced by [this PR](https://github.com/StanGirard/quivr/pull/969), which has already updated the placeholder conveniently